blueprints/screening_app/prediabetes.py

```diff
@@ -0,0 +1,213 @@
+from mindmeld.core import FormEntity
+
+Q_AGE = "1"
+Q_GENDER = "2"
+Q_GEST = "3"
+Q_FAM = "4"
+Q_BP = "5"
+Q_ACT = "6"
+Q_HEIGHT = "7"
+Q_WEIGHT = "8"
+
+LOW_RISK_MSG = ('Su riesgo de padecer de prediabetes es bajo, '
+                'sin embargo sólo un doctor puede darle un diagnóstico confiable.')
+HIGH_RISK_MSG = ('Usted presenta un riesgo elevado de padecer de prediabetes y de '
+                 'desarrollar diabetes tipo 2. Sólo un doctor puede darle un '
+                 'diagnóstico confiable por lo que recomendamos visite a su médico.')
+
+one_pt_weight_cats = {
+    '4\'10"': (119, 142),
+    '4\'11"': (124, 147),
+    '5\'0"': (128, 152),
+    '5\'1"': (132, 157),
+    '5\'2"': (136, 163),
+    '5\'3"': (141, 168),
+    '5\'4"': (145, 173),
+    '5\'5"': (150, 179),
+    '5\'6"': (155, 185),
+    '5\'7"': (159, 190),
+    '5\'8"': (164, 196),
+    '5\'9"': (169, 202),
+    '5\'10"': (174, 208),
+    '5\'11"': (179, 214),
+    '6\'0"': (184, 220),
+    '6\'1"': (189, 226),
+    '6\'2"': (194, 232),
+    '6\'3"': (200, 239),
+    '6\'4"': (205, 245)
```

Comment:
what is the second element of the tuple for? I don't see it used.

Reply:
It's a range. I corrected a mistake in the code that was not using the second value.